vic/drivers/shared_all/src/initialize_veg.c

```diff
@@ -39,35 +39,44 @@ initialize_veg(veg_var_struct **veg_var,
 
     for (i = 0; i < Nveg; i++) {
         for (j = 0; j < options.SNOW_BAND; j++) {
+            // Prognostic states
             veg_var[i][j].albedo = 0.0;
             veg_var[i][j].displacement = 0.0;
             veg_var[i][j].fcanopy = 0.0;
             veg_var[i][j].LAI = 0.0;
             veg_var[i][j].roughness = 0.0;
-            veg_var[i][j].throughfall = 0.0;
             veg_var[i][j].Wdew = 0.0;
             veg_var[i][j].Wdmax = 0.0;
```

Comment:
Personally I'm OK with leaving it as it currently is.  I was just pointing out that Wdmax is derived from LAI - it's good to know this if we want to avoid future problems similar to the one you just fixed (for example, someone who doesn't know it's derived from LAI might change LAI somewhere else in the code and forget to include an update to Wdmax).  I don't know if there is a different way to handle updating Wdmax that is not also cumbersome (since LAI is updated in a few different places).  I suppose one way to handle it could be to eliminate Wdmax from veg_var, veg_con, and veg_lib structures, and instead rewrite the logic that uses Wdmax to compute Wdmax from LAI on the spot immediately before it is used, and for that particular Wdmax to be a temporary variable that exists only within that function.  But that might be too much trouble. @jhamman and @bartnijssen might want to weight in on this.